src/masternode/masternode-sync.cpp

```diff
@@ -372,11 +372,6 @@ void CMasternodeSync::UpdatedBlockTip(const CBlockIndex *pindexNew, bool fInitia
                 pindexNew->nHeight, pindexBestHeader->nHeight, fInitialDownload, fReachedBestHeader);
 
     if (!IsBlockchainSynced() && fReachedBestHeader) {
-        if (fLiteMode) {
-            // nothing to do in lite mode, just finish the process immediately
-            nCurrentAsset = MASTERNODE_SYNC_FINISHED;
-            return;
-        }
```

Comment:
This is not enough, should still skip governance sync in litemode (makes no sense to request objects and then ignore incoming messages), smth like
```diff
diff --git a/src/masternode/masternode-sync.cpp b/src/masternode/masternode-sync.cpp
index d1de7d17ec..800de4e51f 100644
--- a/src/masternode/masternode-sync.cpp
+++ b/src/masternode/masternode-sync.cpp
@@ -230,6 +230,11 @@ void CMasternodeSync::ProcessTick(CConnman& connman)
             // GOVOBJ : SYNC GOVERNANCE ITEMS FROM OUR PEERS
 
             if(nCurrentAsset == MASTERNODE_SYNC_GOVERNANCE) {
+                if (fLiteMode) {
+                    SwitchToNextAsset(connman);
+                    connman.ReleaseNodeVector(vNodesCopy);
+                    return;
+                }
                 LogPrint(BCLog::GOBJECT, "CMasternodeSync::ProcessTick -- nTick %d nCurrentAsset %d nTimeLastBumped %lld GetTime() %lld diff %lld\n", nTick, nCurrentAsset, nTimeLastBumped, GetTime(), GetTime() - nTimeLastBumped);
 
                 // check for timeout first
```

Reply:
331faeb4018cf618e8d805984b40efbbf0b57a62